### PR TITLE
Create li and anchors with images from the specified array.

### PR DESCRIPTION
**Pull request by tom106gunma**
# [DOM構築 4](https://github.com/kenmori/handsonFrontend/blob/master/work/markup/1.md#4)
```js
[{to: "bookmark.html", img: "1.png", alt:"画像1", text: "ブックマーク"}, {to: "message.html", img: "2.png", alt:"画像2", text: "メッセージ"}]
```

という配列を使って以下のようなHTML出力にしてください

```html
<ul>
 <li><a href="/bookmark.html"><img src="1.png" alt="画像1">ブックマーク</a></li>
 <li><a href="/message.html"><img src="2.png" alt="画像2">メッセージ</a></li>
</ul>
```
## Concerns and areas to focus on
-

### CODE REVIEW

**Review by sae-github — APPROVED** [review]

File: lesson04/index.js [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+    img: "2.png",
+    alt:"画像2",
+    text: "メッセージ"
+  }
+];
+
+const fragment = document.createDocumentFragment();
+
+for(const attribute of attributes) {
+  const li = document.createElement("li");
```
Comment (groovywave) [inline lesson04/index.js:19]: LGTM:It's okay.
Why did you change from foreach to for of?
Do you have any specific reason?
Please teach me If you have that.

**Review by groovywave — APPROVED** [review]

**Review by groovywave — APPROVED** [review]

**Review by groovywave — APPROVED** [review]

**Review by groovywave — APPROVED** [review]

**Review by groovywave — APPROVED** [review]

**Review by wattttaw — APPROVED** [review]

File: lesson04/index.js (hunk 1 above)
Comment (tom106gunma) [reply lesson04/index.js:19]: @groovywave 
Thanks for the review.

I wanted to add a change, and it was similar to issue 3.
